rules.d/60-drm.rules

```diff
@@ -3,6 +3,9 @@
 ACTION!="remove", SUBSYSTEM=="drm", SUBSYSTEMS=="pci|usb|platform", IMPORT{builtin}="path_id"
 
 # by-path
-ENV{ID_PATH}=="?*", KERNEL=="card*", SYMLINK+="dri/by-path/$env{ID_PATH}-card"
-ENV{ID_PATH}=="?*", KERNEL=="controlD*", SYMLINK+="dri/by-path/$env{ID_PATH}-control"
-ENV{ID_PATH}=="?*", KERNEL=="renderD*", SYMLINK+="dri/by-path/$env{ID_PATH}-render"
+KERNEL=="card*",     ENV{ID_PATH}=="?*",            SYMLINK+="dri/by-path/$env{ID_PATH}-card"
+KERNEL=="card*",     ENV{ID_PATH_USB_COMPAT}=="?*", SYMLINK+="dri/by-path/$env{ID_PATH_USB_COMPAT}-card"
```

Comment:
mmh will this break existing software? If so, shouldn't it be the other way around, and a new ENV added that includes the protocol revision, leaving the existing unchanged?

Reply:
We discussed this during a call today, and I also think that adding a new variable would be safer.

http://codesearch.debian.net/search?q=ID_PATH&literal=1&perpkg=1 shows that there are some packages which look at ID_PATH. Also, people can have custom unpackaged rules.